rfc6962-bis.xml

```diff
@@ -557,6 +562,9 @@ SEQUENCE {
           <spanx style="verb">key_id</spanx> is the SHA-256 hash of the log's public key, calculated over the DER encoding of the key represented as SubjectPublicKeyInfo.
         </t>
         <t>
+          <spanx style="verb">issuer_key_hash</spanx> is the SHA-256 hash of the certificate issuer's public key, calculated over the DER encoding of the key represented as SubjectPublicKeyInfo. This is needed to bind the issuer to the final certificate.
```

Comment:
More properly, I think, is to "bind the issuer, as defined by OCSP, to the final certificate".

Too lazy to check what CRLs need right now.